src/improvements/tasks/MultisigTask.sol

```diff
@@ -840,8 +840,16 @@ abstract contract MultisigTask is Test, Script, ITask {
         }
     }
 
+    /// @notice helper function that can be overridden by template contracts to
+    /// check the state changes applied by the task. This function can check
+    /// that only the nonce changed in the parent multisig when executing a task
+    /// by checking the slot and address where the slot changed.
+    function checkStateDiff(VmSafe.AccountAccess[] memory) internal view virtual {}
```

Comment:
We should avoid empty default implementations, as they make it too easy to forget to implement safety-critical functions like this (need to use `require(false, msg)` instead of `revert(msg)` to prevent solidity from warning about unreachable code in child contracts
```suggestion
    function checkStateDiff(VmSafe.AccountAccess[] memory) internal view virtual {
      require(false, "checkStateDiff must be implemented");
    }
```

Reply:
Even if for some unexpected reason it makes sense for templates to leave this function empty, it's good to include this revert, so that the template will be forced to implement the empty function body and be reviewed thusly.